svgdigitizer/electrochemistry/cv.py

```diff
@@ -19,110 +19,199 @@
+        >>> str(q.unit)
+        'A / cm2'
 
-        self.modify_df()
+        These string can directly be converted to a unit
 
-    def get_rate(self):  # TODO: probably not required
-        r'''
-        Return rate based on the x coordinate units.
+        >>> u.Unit('uA / cm2')
+        Unit("uA / cm2")
 
-        At the moment we simply use the value.
-        '''
-        self.rate = self.description['scan rate']['value']
-        return self.rate
+        """
+        unit_typos = {'uA / cm2': ['uA / cm2', 'uA / cm²', 'µA / cm²', 'µA cm⁻²', 'uA cm-2', 'uA / cm2'],
+                      'A / cm2': ['A / cm2', 'A cm⁻²', 'A cm-2', 'A / cm2'],
+                      'A': ['A', 'ampere', 'amps', 'amp'],
+                      'mV': ['milliV', 'millivolt', 'milivolt', 'miliv', 'mV'],
+                      'V': ['V', 'v', 'Volt', 'volt'],
+                      'V / s': ['V s-1', 'V/s', 'V / s'],
+                      'mV / s': ['mV / s', 'mV s-1', 'mV/s']}
+
+        for correct_unit, typos in unit_typos.items():
+            for typo in typos:
+                if unit == typo:
+                    return u.Unit(correct_unit)
+
+        raise ValueError(f'Unknown Unit {unit}')
+
+    @property
+    @cache
+    def rate(self):
+        r"""
+        Return the scan rate of the plot.
+
+        The scan rate is read from a `<text>` in the SVG file such as `<text>scan rate: 50 V/s</text>`.
+
+        Examples::
+
+        >>> from svgdigitizer.svg import SVG
+        >>> from svgdigitizer.svgplot import SVGPlot
+        >>> from svgdigitizer.electrochemistry.cv import CV
+        >>> from io import StringIO
+        >>> svg = SVG(StringIO(r'''
+        ... <svg>
+        ...   <g>
+        ...     <path d="M 0 200 L 0 100" />
+        ...     <text x="0" y="200">x1: 0 cm</text>
+        ...   </g>
+        ...   <g>
+        ...     <path d="M 100 200 L 100 100" />
+        ...     <text x="100" y="200">x2: 1cm</text>
+        ...   </g>
+        ...   <g>
+        ...     <path d="M -100 100 L 0 100" />
+        ...     <text x="-100" y="100">y1: 0</text>
+        ...   </g>
+        ...   <g>
+        ...     <path d="M -100 0 L 0 0" />
+        ...     <text x="-100" y="0">y2: 1 A</text>
+        ...   </g>
+        ...   <text x="-200" y="330">scan rate: 50 V/s</text>
+        ... </svg>'''))
+        >>> cv = CV(SVGPlot(svg))
+        >>> cv.rate
+        <Quantity 50. V / s>
 
-    def modify_df(self):
-        # Create potential columns
-        self.df = self.create_df_U_axis(self.svgplot.dfs[0])
-        # Create current columns
-        self.df = pd.concat(
-            [self.df, self.create_df_I_axis(self.svgplot.dfs[0])],
-            axis=1)
+        """
+        rates = self.svgplot.svg.get_texts('(?:scan rate|rate): (?P<value>-?[0-9.]+) *(?P<unit>.*)')
+        # To Do: assert that only one label contains the scan rate (see issue #58)
+        # To Do: assert that a rate is available at all (see issue #58)
```

Comment:
```suggestion
        # TODO: assert that only one label contains the scan rate (see issue #58)
        # TODO: assert that a rate is available at all (see issue #58)
```
Can we always write `TODO`? I think that this is the format that most editors highlight specially.